roles/openshift_logging_curator/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
 openshift_logging_curator_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_curator_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_curator_image_version: "{{ openshift_logging_image_version }}"
```

Comment:
We need to be able to define specific component versions for 3.6 to avoid the upgrade issues we see by customers defining generic versions like 'v3.5'.  IMO, the price to pay for removing 'hosted' vars is better then having to spend time debugging why a log stack is not working because a restarted pod unintentionally pulled in a new version.